include/slb/type_traits.hpp

```diff
@@ -506,6 +503,28 @@ template <typename T>
 struct is_rvalue_reference
     : slb::bool_constant<std::is_rvalue_reference<T>::value> {};
 
+template <typename T>
+struct is_enum : slb::bool_constant<std::is_enum<T>::value> {};
+
+template <typename T>
+struct is_union : slb::bool_constant<std::is_union<T>::value> {};
+
+template <typename T>
+struct is_class : slb::bool_constant<std::is_class<T>::value> {};
+#endif
+
+// libstdc++ `is_function` does not recognize `noexcept` until version 7.
+#if !defined(__GLIBCXX__) || __has_include(<variant>) // >= libstdc++-7
+#define SLB_IS_FUNCTION SLB_INTEGRAL_CONSTANT
+#else
+#define SLB_IS_FUNCTION 0 // not available
+#endif
+
+#if SLB_IS_FUNCTION == 2
+using std::is_member_object_pointer;
+using std::is_member_function_pointer;
+using std::is_function;
+#elif SLB_TRIVIALITY_TRAITS == 1
```

Comment:
Is this really supposed to be `SLB_TRIVIALITY_TRAITS` here?

Reply:
Obviously not, just bad copy/paste gone bad. Good catch.